modules/policy-criteria.adoc

```diff
@@ -29,524 +30,831 @@ In this table:
+*Runtime* (when used with a Runtime criterion)
+
+| Runtime Class
+| The RuntimeClass of the deployment.
+| Runtime Class
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Host Network
+| Check if `HostNetwork` is enabled which means that the container is not placed inside a separate network stack (for example, the container's networking is not containerized). This implies that the container has full access to the host's network interfaces.
+| Host Network
+| Boolean
+| ✕
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Host PID
+| Check if the Process ID (PID) namespace is isolated between the containers and the host. This allows for processes in different PID namespaces to have the same PID.
+| Host PID
+| Boolean
+| ✕
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Host IPC
+| Check if the IPC (POSIX/SysV IPC) namespace (which provides separation of named shared memory segments, semaphores and message queues) on the host is shared with containers.
+| Host IPC
+| Boolean
 | ✕
-| ✓
-| Deploy
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Namespace
+| The name of the namespace the deployment belongs to.
+| Namespace
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Replicas
+| The number of deployment replicas.
+| Replicas
+| <, >, <=, >= or nothing (which implies equals to) +
+
+-- and --
+ +
+a decimal (a number with an optional fractional value). +
+
+Examples: +
+>=5, or +
+9.5
+| NOT, +
+AND, OR
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+6+| *Section: Storage*
 
 | Volume Name
 | Name of the storage.
-| All
-| ✓
-| ✓
-| ✓
-| Deploy
+| Volume Name
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
 
 | Volume Source
 | Indicates the form in which the volume is provisioned. For example, `persistentVolumeClaim` or `hostPath`.
-| All
-| ✓
-| ✓
-| ✓
-| Deploy
+| Volume Source
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
 
 | Volume Destination
 | The path where the volume is mounted.
-| All
-| ✓
-| ✓
-| ✓
-| Deploy
+| Volume Destination
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
 
 | Volume Type
 | The type of volume.
-| All
-| ✓
-| ✓
-| ✓
-| Deploy
-
-| Writable Volume
+| Volume Type
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Mounted volume writability
 | Volumes that are mounted as writable.
-| All
-| ✕
-| ✕
+| Writable Mounted Volume
+| Boolean
+| ✕
+| *Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Mount Propagation
+| Check if container is mounting volumes in `Bidirectional`, `Host to Container`, or `None` modes.
+| Mount Propagation
+| One of: +
+
+NONE +
+HOSTTOCONTAINER +
+BIDIRECTIONAL +
+| ✕NOT, +
```

Comment:
Should the `x` here be removed?

Reply:
I assume it's here because regex is not supported, so maybe a comma between the `x` and `NOT`? I kept skipping over it thinking it read "XNOR" and only just realized that it's not.